src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/configure-entries.jelly

```diff
@@ -27,6 +27,10 @@
             <f:select/>
         </f:entry>
 
+        <f:entry title="Max number of pipelines" field="maxNumberOfJobs" default="-1">
+            <f:textbox/>
+        </f:entry>
```

Comment:
Is it possible to only show this in the UI if sort by latest activity is selected?